src/redux/reducers/ui.js

```diff
@@ -10,6 +10,9 @@ const defaultState = {
     isSnackBarOpen: false,
     snackBarMessage: '',
     snackBarDuration: 2500,
+    snackBarPosition: { vertical: 'bottom', horizontal: 'center' },
+    variant: null,
```

Comment:
variant cannot be null, since `variant: PropTypes.oneOf(['success', 'warning', 'error']).isRequired,`. Change it to success.